### PR TITLE
Add blog post about ManageIQ RbVmomi maintenance

### PR DESCRIPTION
**Pull request by agrare**

### CODE REVIEW

File: site/_posts/2021-09-15-rbvmomi-maintenance.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+I have the pleasure of announcing that the [ManageIQ community](https://www.manageiq.org/community/) will be taking over ongoing maintenance of the RbVmomi gem.
+
+RbVmomi is a Ruby SDK for the VMware vSphere API which allows users to manage their virtual infrastructure.
+
+Up until this point RbVmomi has been managed by VMware as a community supported project, but due to lack of Ruby maintainers it has since been archived [[ref]](https://github.com/vmware-archive/rbvmomi/commit/fb3caa94264827c27f00a7369090bd2421b82524).
+
+The ManageIQ community is picking up where VMware left off and will be providing security updates, bug fixes, and updates for new versions of VMware vSphere in a timely manner.
+
+The gem will continue to be available from rubygems at https://rubygems.org/gems/rbvmomi so if you are currently using RbVmomi you don't need to change anything.
```
Comment (agrare) [inline site/_posts/2021-09-15-rbvmomi-maintenance.md:16]: TBD pending gem owner access

**Review by chessbyte — APPROVED** [review]

File: site/_posts/2021-09-15-rbvmomi-maintenance.md (hunk 1 above)
Comment (agrare) [reply site/_posts/2021-09-15-rbvmomi-maintenance.md]: VMware has decided to not allow for the continued use of https://rubygems.org/gems/rbvmomi

File: site/_posts/2021-09-15-rbvmomi-maintenance.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+title: RbVmomi Maintenance
+date: 2021-09-15 18:00 UTC
+author: agrare
+tags: collaboration announcements
+---
+
+I have the pleasure of announcing that the [ManageIQ community](https://www.manageiq.org/community/) will be taking over ongoing maintenance of the RbVmomi gem.  ManageIQ it is an Open Source Hybrid-Cloud Automation and Management Platform with a large number of supported Clouds, Virtual and Container environments, and more.
```
Comment (chessbyte) [inline site/_posts/2021-09-15-rbvmomi-maintenance.md:8]:
```suggestion
I have the pleasure of announcing that the [ManageIQ community](https://www.manageiq.org/community/) will be taking over ongoing maintenance of the RbVmomi gem.  ManageIQ is an Open Source Hybrid-Cloud Automation and Management Platform with a large number of supported Clouds, Virtual and Container environments, and more.
```

Comment (agrare) [reply site/_posts/2021-09-15-rbvmomi-maintenance.md]: :+1: thanks, overly edited :)

File: site/_posts/2021-09-15-rbvmomi-maintenance.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+title: RbVmomi Maintenance
+date: 2021-09-15 18:00 UTC
+author: agrare
+tags: collaboration announcements
+---
+
+I have the pleasure of announcing that the [ManageIQ community](https://www.manageiq.org/community/) will be taking over ongoing maintenance of the RbVmomi gem.  ManageIQ is an Open Source Hybrid-Cloud Automation and Management Platform with a large number of supported Clouds, Virtual and Container environments, and more.
```
Comment (Fryguy) [inline site/_posts/2021-09-15-rbvmomi-maintenance.md:8]:
```suggestion
I have the pleasure of announcing that the [ManageIQ community](https://www.manageiq.org/community/) will be taking over ongoing maintenance of the RbVmomi gem.  ManageIQ is an Open Source Hybrid-Cloud Automation and Management Platform with a large number of supported Cloud, Virtual and Container environments, and more.
```

File: site/_posts/2021-09-15-rbvmomi-maintenance.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+Initially the gem maintainers will be [Adam Grare](https://github.com/agrare) and [Jason Frey](https://github.com/Fryguy).  The ManageIQ project has been using RbVmomi for its VMware vSphere provider [manageiq-providers-vmware](https://github.com/ManageIQ/manageiq-providers-vmware) for years. Jason is a staple of the Ruby and Rails communities, and Adam has been an active contributor to RbVmomi since 2016.
+
+The gem will be available from rubygems at https://rubygems.org/gems/rbvmomi2. If you are currently using RbVmomi you will need to update your Gemfile or gemspec and replace `rbvmomi` with `rbvmomi2`.
+
+The new repository is https://github.com/ManageIQ/rbvmomi2; as always we encourage the community to open issues and contribute code and ideas.  If you are interested in contributing check out https://github.com/ManageIQ/rbvmomi/blob/master/CONTRIBUTING.md and open an [Issue](https://github.com/ManageIQ/rbvmomi/issues/new/choose) or [Pull Request](https://github.com/ManageIQ/rbvmomi/compare), or just discuss something in the [RbVmomi Gitter room](https://gitter.im/vmware/rbvmomi).
```
Comment (Fryguy) [inline site/_posts/2021-09-15-rbvmomi-maintenance.md:20]:
```suggestion
The new repository is https://github.com/ManageIQ/rbvmomi2; as always we encourage the community to open issues and contribute code and ideas.  If you are interested in contributing, check out https://github.com/ManageIQ/rbvmomi/blob/master/CONTRIBUTING.md and open an [Issue](https://github.com/ManageIQ/rbvmomi/issues/new/choose) or [Pull Request](https://github.com/ManageIQ/rbvmomi/compare), or just discuss something in the [RbVmomi Gitter room](https://gitter.im/vmware/rbvmomi).
```

File: site/_posts/2021-09-15-rbvmomi-maintenance.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+And if you're new to ManageIQ, welcome and check us out https://github.com/ManageIQ and the ManageIQ Gitter room https://gitter.im/ManageIQ/manageiq !
```
Comment (Fryguy) [inline site/_posts/2021-09-15-rbvmomi-maintenance.md:22]:
```suggestion
If you're new to ManageIQ, welcome! Come check out https://manageiq.org, see the [source code](https://github.com/ManageIQ) and chat with us in the [ManageIQ Gitter room](https://gitter.im/ManageIQ/manageiq) !
```

File: site/_posts/2021-09-15-rbvmomi-maintenance.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
+---
+title: RbVmomi Maintenance
+date: 2021-09-21 18:00 UTC
```
Comment (Fryguy) [inline site/_posts/2021-09-15-rbvmomi-maintenance.md:3]: This needs to be updated once we actually get to posting it.

**Review by Fryguy — CHANGES_REQUESTED** [review]

File: site/_posts/2021-09-15-rbvmomi-maintenance.md (hunk 6 above)
Comment (Fryguy) [reply site/_posts/2021-09-15-rbvmomi-maintenance.md:3]: Incidentally, you don't need the time...the date is sufficient.

Comment (agrare) [reply site/_posts/2021-09-15-rbvmomi-maintenance.md]: That's interesting, a lot of posts have the time :shrug e.g. `site/_posts/2021-04-26-Announcing-Lasker-Beta.md:date: 2021-04-26 13:27:43 UTC`

Updated date and removed time

File: site/_posts/2021-12-21-rbvmomi-maintenance.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+Up until this point RbVmomi has been managed by VMware as a community supported project, but due to a lack of Ruby maintainers it has since been archived [[ref]](https://github.com/vmware-archive/rbvmomi/commit/fb3caa94264827c27f00a7369090bd2421b82524).
+
+The ManageIQ community is picking up where VMware left off and will be providing security updates, bug fixes, and updates for new versions of VMware vSphere in a timely manner.
+
+Initially the gem maintainers will be [Adam Grare](https://github.com/agrare) and [Jason Frey](https://github.com/Fryguy).  The ManageIQ project has been using RbVmomi for its VMware vSphere provider [manageiq-providers-vmware](https://github.com/ManageIQ/manageiq-providers-vmware) for years. Jason is a staple of the Ruby and Rails communities, and Adam has been an active contributor to RbVmomi since 2016.
+
+The gem will be available from rubygems at https://rubygems.org/gems/rbvmomi2. If you are currently using RbVmomi you will need to update your Gemfile or gemspec and replace `rbvmomi` with `rbvmomi2`.
```
Comment (chessbyte) [inline site/_posts/2021-12-21-rbvmomi-maintenance.md:16]:
```suggestion
The gem is available from rubygems at https://rubygems.org/gems/rbvmomi2. If you are currently using RbVmomi you will need to update your Gemfile or gemspec and replace `rbvmomi` with `rbvmomi2`.
```

Comment (agrare) [reply site/_posts/2021-12-21-rbvmomi-maintenance.md]: Done

**Comment by agrare** [comment]
Okay should be ready @chessbyte @Fryguy PTAL

**Pull request merged**